.pre-commit-config.yaml

```diff
@@ -23,6 +23,11 @@ repos:
         language_version: python3
     -   id: flake8
         language_version: python3
+-   repo: https://github.com/asottile/reorder_python_imports
+    rev: v1.3.2
+    hooks:
+    -   id: reorder-python-imports
+        args: ['--application-directories=.:src']
```

Comment:
I should probably also set `--add-import 'from __future__ import absolute_import'` and/or the others we want to enforce on all files